### PR TITLE
Move more contents from Vagrantfile to Dockerfile

### PR DESCRIPTION
**Pull request by tobsan**
Please do not merge before the CI has run.

As we are transitioning away from using Vagrant, there are now only
settings for synced folders and building the docker image left in the
Vagrantfile. These can probably be moved to be done by Jenkins later on.

Signed-off-by: Tobias Olausson <tobias.olausson@pelagicore.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,28 +1,60 @@
...
 
 USER root
 
-RUN mkdir /var/run/sshd
-
 # Install dependencies in one command to avoid potential use of previous cache
 # like explained here: https://stackoverflow.com/a/37727984
-RUN apt-get update \
-    && apt-get install -y openssh-server inetutils-ping iptables cvs subversion coreutils python3-pip libfdt1 python-pysqlite2 help2man libxml2-utils libsdl1.2-dev graphviz qemu-user g++-multilib curl
+RUN apt-get update && apt-get install -y \
```
Comment (GordanM) [inline Dockerfile]: Unrelated change.

Comment (tobsan) [reply Dockerfile]: True that. Would you rather see it separated?

Comment (GordanM) [reply Dockerfile]: Yeah that would be good, because applying patches (if it comes to that) might prove to be tricky later on.

Comment (GordanM) [reply Dockerfile]: Thanks!

File: Dockerfile [hunk 2]
```diff
@@ -1,28 +1,60 @@
...
+        python3-pip \
+        libfdt1 \
+        python-pysqlite2 \
+        help2man \
+        libxml2-utils \
+        libsdl1.2-dev \
+        graphviz \
+        qemu-user \
+        g++-multilib \
+        curl
```
Comment (DunderRoffe) [inline Dockerfile]: Add a \ at the end of this row as well, then you can mix and match easier

Comment (tobsan) [reply Dockerfile]: Adding a backslash there makes the Dockerfile unparseable, so no.

Comment (tobsan) [reply Dockerfile]: Or rather, it gives a warning that will become an error in future versions.

Comment (DunderRoffe) [reply Dockerfile]: :/
